backend/graphql/typeDefs.js

```diff
@@ -11,19 +11,52 @@ module.exports = gql`
     createdAt: String!
     verified: Boolean!
   }
+
+  type Listing {
+    id: ID!
+    title: String!
+    price: Float!
+    numberOfRoommates: Int!
+    bathroomType: String!
+    location: String!
+    leaseDurationMonths: Int!
+    isFurnished: Boolean!
+    utilitiesIncluded: Boolean!
+    petsAllowed: Boolean!
+    createdAt: String!
```

Comment:
If there's user in the model, shouldn't it also be here?

Reply:
I can add that

Reply:
let us know when you fix it and I can approve to merge it!